subsys/partition_manager/Kconfig

```diff
@@ -141,8 +141,8 @@ endif
 config BUILD_WITH_TFM
 	bool
 	select PM_SINGLE_IMAGE
-	select ENTROPY_CC3XX
-	select NORDIC_SECURITY_BACKEND
+	select ENTROPY_CC3XX if SOC_FAMILY_NRF
+	select NORDIC_SECURITY_BACKEND if SOC_FAMILY_NRF
```

Comment:
I placed comments in the relevant commits where this should be cleaned up, in which case this commit (9a0fec05b32a963da3fe88c7db9a1eefbcb7f3d8) can be dropped.

Reply:
Will fix (cleanup before run)

Reply:
Fixed